source/manual/2nd-line-drills.html.md

```diff
@@ -92,3 +93,48 @@ Either before merging the change, or part way through the continuous deployment
 Follow the deployment pipeline in Jenkins. Confirm that no further environment deployments are triggered. For example, if you implemented the deploy freeze just after the app was deployed to Staging, confirm that the app was then not automatically deployed to Production.
 
 Remove the code freeze, then manually push the changes to all remaining environments so that they're in sync.
+
+## Drill getting a password out of govuk-secrets
+
+Most of our shared accounts are stored in this repo. Follow the steps to [get a password](https://github.com/alphagov/govuk-secrets/tree/main/pass#get-a-password), a useful one for 2nd line is the [Technical 2nd line Dashboard](https://alphagov.github.io/frame-splits/index.html?title=2nd+Line+Dashboard&layout=2x1-responsive&url%5B%5D=https%3A%2F%2Fgovuk-secondline-blinken.herokuapp.com%2Fblinken.html&url%5B%5D=https%3A%2F%2Fgrafana.production.govuk.digital%2Fdashboard%2Ffile%2F2ndline_health.json&url%5B%5D=https%3A%2F%2Fgovuk-zendesk-display-screen.herokuapp.com&url%5B%5D=) which shows a high level overview of the state of GOV.UK environments.
+Upon loading this dashboard it will ask for the `govuk-zendesk-display-screen` credentials.
+
+## Drill assigning a user to their publisher in data.gov.uk
+
+[Log into our shared data.gov.uk publisher account](/manual/data-gov-uk-2nd-line.html#logging-into-the-publisher). Pick a [publisher](https://ckan.publishing.service.gov.uk/organization) to do a hypothetical walk though of [Assign users to publishers](/manual/data-gov-uk-2nd-line.html#assign-users-to-publishers-setting-user-permissions).
+
+## Drill redirecting a route
+
+Follow the [Removing a route created in the Short URL Manager](/manual/redirect-routes.html#removing-a-route-created-in-the-short-url-manager) and [Removing a route completely so it can be replaced with another route](/manual/redirect-routes.html#removing-a-route-completely-so-it-can-be-replaced-with-another-route) instructions.
+Do this on Integration or Staging.
+
+## Drill changing a slug and create a redirect
+
+Follow the [Change a slug and create redirect in Whitehall](/manual/howto-change-slug-and-create-redirect.html), picking something
+at random in Whitehall from one of the group of entities listed ([people](https://whitehall-admin.publishing.service.gov.uk/government/admin/people), [role](https://whitehall-admin.publishing.service.gov.uk/government/admin/roles), [organisation](https://whitehall-admin.publishing.service.gov.uk/government/admin/organisations), etc).
+Do this on Integration or Staging.
+
+## Drill publishing a document
+
+Create a new document by following the GOV.UK guidance on how to [create a new document](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#create-a-new-document) and [preview](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#preview-content) it.
+Do this on Integration or Staging.
```

Comment:
This feels a bit unnecessary in hindsight - I've never had to publish a document for my 2nd line or on-call shifts. I think publishers have enough permissions here that they never need technical assistance for this task.

Reply:
Thanks @ChrisBAshton ! So I'd added a drill for this because a couple of the devs doing their shadow shift mentioned they'd not had to interact with Whitehall before, so some sort of introduction into how it works would be helpful. I'm happy to leave this one out and maybe add it as a specific task in the Shadow Trello template card?

Reply:
That sounds better to me 👍